.github/scripts/forc-handler.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+PROJECT=$1
+COMMAND=$2
+FLAGS=$3
+
+if [ $PROJECT = 'airdrop/airdrop-distributor' ]; then
+    forc $COMMAND --path $PROJECT $FLAGS
+    forc $COMMAND --path $PROJECT/../simple-asset/ $FLAGS
+elif [ $PROJECT = 'airdrop/simple-asset' ]; then
+    forc $COMMAND --path $PROJECT $FLAGS
+elif [ $PROJECT = 'auctions/english-auction' ]; then
+    forc $COMMAND --path $PROJECT $FLAGS
+    forc $COMMAND --path $PROJECT/tests/artifacts/asset $FLAGS
+    forc $COMMAND --path $PROJECT/../../NFT/ $FLAGS
+elif [ $PROJECT = 'dao-voting' ]; then
+    forc $COMMAND --path $PROJECT $FLAGS
+    forc  $COMMAND --path $PROJECT/tests/artifacts/gov_token $FLAGS
```

Comment:
```suggestion
    forc $COMMAND --path $PROJECT/tests/artifacts/gov_token $FLAGS
```

Reply:
I was able to get the script to work locally with multiple `FLAGS` by setting `FLAGS="${@:3}"` but it didn't seem to work in github and after roughly an hour of debugging it still doesn't and I don't know why.  Because of that and seeing as no command in the ci requires multiple flags I'm just gonna make it work with a single flag.